code/modules/chemistry/Chemistry-Tools.dm

```diff
@@ -894,6 +894,22 @@ proc/ui_describe_reagents(atom/A)
 		..()
 
 	attackby(var/obj/item/W, mob/user)
+		if(istype(W, /obj/item/reagent_containers/iv_drip))
+			var/obj/item/reagent_containers/iv_drip/iv = W
+			if(!iv.slashed)
+				boutput(user, "<span class='alert'>The [iv.name] needs to be cut open first!</span>")
+				return
+			else if (reagents.total_volume >= reagents.maximum_volume)
```

Comment:
would using [trans_to](https://github.com/goonstation/goonstation/blob/a6950c7e5705a4d4973a3262a27cbd59fde6a41a/code/modules/chemistry/Chemistry-Holder.dm#L295) on the IV's reagent holder cause any trouble? It does some of these checks too

Reply:
The reason I made it add chems this way instead of doing it directly is because currently all of the checks pustules do to play sounds/reject chems is on reagent_act(), which is triggered by pouring stuff on it. It's probably be better to rewrite that code to work with trans_to and then just call that on reagent_act though in hindsight I guess? Just swapping it to trans_to as is though would let you pour stuff into pustules you shouldn't be able to though